blocks/leadspace/leadspace.css

```diff
@@ -7,6 +7,16 @@
     letter-spacing: var(--letter-spacing-1);
     }
 
+.leadspace div.only-picture, .leadspace p.only-picture {
```

Comment:
Add newline after `,`.

`only-picture` is a new Leadspace variant?

Reply:
Yes, in the JS.